src/test/java/uk/gov/ch/developer/docs/controller/developer/SignOutControllerTest.java

```diff
@@ -39,87 +34,21 @@
     private HttpServletResponse httpServletResponse;
     @Mock
     private Session session;
+    @Mock
+    private Oauth2 oauth2;
 
     private final SignInInfo signInInfo = new SignInInfo();
 
     @InjectMocks
     private SignOutController signOutController;
 
     @Test
-    @DisplayName("Test that a valid signed in user's session state is correctly altered")
-    public void testDoSignOutWhenSignedIn() throws IOException {
-        final String zxsValue = "0000000001z";
-
-        signInInfo.setSignedIn(true);
-
+    @DisplayName("Test that a user is redirected upon signing out")
+    public void testDoSignOut() throws IOException {
```

Comment:
I think we ought to verify that the invalidate method it called.

Reply:
I have added a new test above the testDoSignOut that verifies that invalidateSession is called.

Reply:
Great that will resolve my concerns above I assume.